storybook/storybook-css/src/components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,197 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Link, Paragraph } from '@amsterdam/design-system-react/src'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/link/link.scss'
+import '@amsterdam/design-system-css/src/paragraph/paragraph.scss'
+import '@amsterdam/design-system-css/src/icon/icon.scss'
+
+type Story = StoryObj<typeof Link>
+
+const meta = {
+  title: 'CSS Components/Link',
+  id: 'link',
+  component: Link,
+  argTypes: {
+    onBackground: {
+      control: { type: 'select', labels: { default: 'default', light: 'light', dark: 'dark' } },
+      options: ['default', 'light', 'dark'],
+      table: {
+        category: 'API',
+        defaultValue: { summary: 'default' },
+      },
+    },
+    variant: {
+      control: {
+        type: 'select',
+        labels: { standalone: 'standalone', standaloneBold: 'standaloneBold', inline: 'inline', inList: 'inList' },
+      },
+      options: ['standalone', 'standaloneBold', 'inline', 'inList'],
+      table: {
+        category: 'API',
+        defaultValue: { summary: 'standalone' },
+      },
+    },
+    href: {
+      name: 'href',
+      type: { name: 'string', required: false },
+      table: {
+        category: 'HTML attribute',
+        defaultValue: { summary: '' },
+      },
+    },
+    rel: {
+      name: 'rel',
+      type: { name: 'string', required: false },
+      table: {
+        category: 'HTML attribute',
+        defaultValue: { summary: '' },
+      },
+    },
+    target: {
+      name: 'target',
+      type: { name: 'string', required: false },
+      table: {
+        category: 'HTML attribute',
+        defaultValue: { summary: '' },
+      },
+    },
+  },
```

Comment:
Do we need to add controls for this native stuff?

Reply:
Added by Utrecht to storybook, I guess for testing or to have complete documentation?